src/software/jetson_nano/services/motor.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#include <memory>
+#include <string>
+
+#include "proto/robot_status_msg.pb.h"
+#include "proto/tbots_software_msgs.pb.h"
+#include "shared/robot_constants.h"
+#include "software/jetson_nano/services/service.h"
+
+class MotorService : public Service
+{
+   public:
+    /**
+     * Service that interacts with the motor board.
+     * Opens all the required ports and maintains them until destroyed.
+     *
+     * @param RobotConstants_t The robot constants
+     * @param WheelConstants_t The wheel constants
+     */
+    MotorService(const RobotConstants_t& robot_constants,
+                 const WheelConstants_t& wheel_constants);
+
+    virtual ~MotorService();
+
+    /**
+     * Starts the motor service by pulling the enable pin high.
+     */
+    void start() override;
```

Comment:
resources are acquired on initialization (i.e spi + gpio file descripters are opened with the right settings)

this doesn't fall under RAII because its going to be used to arm/disarm the motor board after we receive the first set of primitives